apollo-portal/src/main/resources/static/i18n/zh-CN.json

```diff
@@ -288,7 +289,9 @@
   "Component.Publish.AllPublish": "全量发布",
   "Component.Publish.AllPublishTips": "（全量发布的配置会作用于全部的实例）",
   "Component.Publish.ToSeeChange": "查看变更",
-  "Component.Publish.PublishedValue": "发布的值",
+  "Component.Publish.change1": "与主版本的值对比",
```

Comment:
These fields are not defined in apollo-portal/src/main/resources/static/i18n/en.json